README.md

```diff
@@ -1,94 +1,93 @@
     json doc4 = { "name": "test", "type": "artifacts" };
-    json filter = { "type": "src" };
-    json replaceFilter = { "type": "artifacts" };
 
-    var ret = conn->insert("projects", doc1);
+    log:printInfo("------------------ Inserting Data -------------------");
+    var ret = mongoClient->insert("projects", doc1);
     handleInsert(ret, "Insert to projects");
-    ret = conn->insert("projects", doc2);
+    ret = mongoClient->insert("projects", doc2);
     handleInsert(ret, "Insert to projects");
-    ret = conn->insert("projects", doc3);
+    ret = mongoClient->insert("projects", doc3);
     handleInsert(ret, "Insert to projects");
-
-    var jsonRet = conn->find("projects", ());
+  
+    log:printInfo("------------------ Querying Data -------------------");
+    var jsonRet = mongoClient->find("projects", ());
     handleFind(jsonRet);
 
-    json queryString = { "name": "ballerina" };
-    jsonRet = conn->find("projects", queryString);
+    json queryString = {name: "connectors" };
+    jsonRet = mongoClient->find("projects", queryString);
     handleFind(jsonRet);
 
-    jsonRet = conn->findOne("projects", queryString);
-    handleFind(jsonRet);
+    json jsonRetOne = mongoClient->findOne("projects", queryString);
+    handleFind(jsonRetOne);
 
-    var deleteRet = conn->delete("projects", filter, true);
-    if (deleteRet is int) {
-        io:println("deleted count: " + deleteRet);
+    log:printInfo("------------------ Updating Data -------------------");
+    json replaceFilter = { "type": "artifacts" };
+    json doc5 = { "name": "main", "type": "artifacts" };
+    boolean upsert = true;
+
+    int response = mongoClient->replace("projects", replaceFilter, doc5,upsert);
+    if (response > 0 ) {
+        log:printInfo("Modified count: ") ;
+        log:printInfo(response.toString());
     } else {
-        io:println("delete failed: " + deleteRet.reason());
+        log:printInfo("Error in replacing data");
     }
 
-    json doc5 = { "name": "main", "type": "artifacts" };
-    var response = conn->replaceOne("projects", replaceFilter, doc5);
-    if (response is int) {
-        io:println("Modified count: " + response);
+   log:printInfo("------------------ Deleting Data -------------------");
+   json deleteFilter = { "name": "ballerina" };
+   var deleteRet = mongoClient->delete("projects", deleteFilter, true);
+    if (response > 0 ) {
+        log:printInfo("Modified count: ") ;
+        log:printInfo(response.toString());
     } else {
-        io:println("Modified count: " + response.reason());
+        log:printInfo("Error in replacing data");
     }
-    conn.stop();
+    
+     mongoClient.stop();
 }
 
-function handleInsert(()|error returned, string message) {
-    if (returned is ()) {
-        io:println(message + " success ");
+function handleInsert(json|error returned, string message) {
+    if (returned is error) {
+        io:println(message + " failed: " , returned.reason());
```

Comment:
Better to change `io:println` to `log:printInfo`

Reply:
Fixed